idaplugin/rematch/updater.py

```diff
@@ -18,6 +18,11 @@ def update():
   if not config['settings']['update']['autocheck']:
     return
 
+  if not config['login']['server']:
+    logger('updater').debug("skipping auto update because server is\
+                            not configured yet, is it your first time ?")
+    return
+
```

Comment:
We can remove it but a nice mechanism of "Is this your first time using REmatch?" should be nice,
maybe not for first release if it's too much burden